e2e/paths/loginAsHmctsAdmin_test.js

```diff
@@ -23,3 +23,8 @@ xScenario('HMCTS admin can login and add a FamilyMan case number to a submitted
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.addFamilyManCaseNumber);
 });
+
+xScenario('HMCTS admin can login and see case name', (I, loginPage, createCasePage) => {
```

Comment:
The problem with the test is that it relies on data entered by other tests which makes it fragile. I suggest removing it at this stage.